types/mapbox__mapbox-gl-draw/index.d.ts

```diff
@@ -0,0 +1,32 @@
+// Type definitions for @mapbox/mapbox-gl-draw 1.1
+// Project: https://github.com/mapbox/mapbox-gl-draw#readme
+// Definitions by: Ryan Vanbelkum <https://github.com/ryanvanbelkum>
+//                 Nathan Weber <https://github.com/nweber-gh>
+//                 Bryan Wain <https://github.com/bdwain>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+declare module "@mapbox/mapbox-gl-draw/src/constants" {
+  enum modes {
```

Comment:
There are basically no times when writing an `enum` declaration in DefinitelyTyped is correct. These should be represented as object types. (Incidentally, you could actually just copy the entire constants.js file and replace all the `=` with `:` to get the correct types.)